atc/worker/streamer.go

```diff
@@ -136,6 +140,19 @@ func (s Streamer) p2pStream(ctx context.Context, src runtime.P2PVolume, dst runt
 		return err
 	}
 
+	// Verify stream-in url
+	rawUrl, err := url.Parse(streamInUrl)
+	if err != nil {
+		return fmt.Errorf("invalid stream-in-url: %w", err)
+	}
+	// If stream limit is set, append the limit to stream-in url
+	if s.limitInMB > 0.00000094 {
```

Comment:
Is there a specific reason to use this number other than `0.0`?

Reply:
I just want the limit to be bigger than 1 byte. I just updated the code to look clearer.